packages/react-components/react-breadcrumb/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -4,15 +4,17 @@ import { renderBreadcrumb_unstable } from './renderBreadcrumb';
 import { useBreadcrumbStyles_unstable } from './useBreadcrumbStyles';
 import type { BreadcrumbProps } from './Breadcrumb.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useBreadcrumbContextValue } from './useBreadcrumbContextValue';
 
 /**
  * Breadcrumb component - TODO: add more docs
  */
 export const Breadcrumb: ForwardRefComponent<BreadcrumbProps> = React.forwardRef((props, ref) => {
   const state = useBreadcrumb_unstable(props, ref);
+  const breadcrumbContextValue = useBreadcrumbContextValue(state);
```

Comment:
```suggestion
  const contextValues = useBreadcrumbContextValues_unstable(state);
```

nit, we use `contextValues` as a suffix. Also hooks should be marked as unstable 😉